sensor_msgs/Image image
int32[] ids
geometry_msgs/Twist[] poses